[Windows] gradlew extractAllJars fails due to invalid symbols in file paths

Windows only issue. Steps to reproduce:

Set a library with `testCompileOnly  `
```
testCompileOnly  "com.android.support:support-v4:27.0.1"
```
Execute 

```
$ ./gradlew extractAllJars
```

Result:
```
FAILURE: Build failed with an exception.

* Where:
Build file 'D:\repositories\android-dts-generator\dts-generator\build.gradle' line: 62

* What went wrong:
Execution failed for task ':extractAllJars'.
> Illegal char <:> at index 45: jar-files\support-v4.aar (com.android.support:support-v4:27.0.1)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
1 actionable task: 1 executed
```

--- a/dts-generator/build.gradle
+++ b/dts-generator/build.gradle
@@ -64,7 +64,7 @@ task extractAllJars {
             //declaring variable as specific class for getting code completion in Android Studio
             org.gradle.api.internal.artifacts.DefaultResolvedArtifact nextDependency = iter.next()
 
-            def outputDir = java.nio.file.Paths.get(extractedDependenciesDir, nextDependency.toString()).normalize().toString()
+            def outputDir = java.nio.file.Paths.get(extractedDependenciesDir, nextDependency.toString()).normalize().toString().replace(':', '_')
             explodeAar(nextDependency.file, outputDir)
             dependencyCounter++
         }

